test-gui-e2e-teardown.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+ARTIFACT_DIR=/tmp/artifacts
+export ARTIFACT_DIR
+
+cp -r gui_test_screenshots "${ARTIFACT_DIR}/gui_test_screenshots"
```

Comment:
I think this will fail in Jenkins since the directory won't exist. It's not gating merges, but still can be run (e.g., with `jenkins push`)

If we knowingly break Jenkins, we should probably remove the jenkins.sh script, too.

Reply:
i could add a prow argument to the script and only run it for prow tests?